src/subprocess-win32.cc

```diff
@@ -74,7 +74,52 @@ HANDLE Subprocess::SetupPipe(HANDLE ioport) {
   return output_write_child;
 }
 
-bool Subprocess::Start(SubprocessSet* set, const string& command) {
+static DWORD POSIXPriorityToWin32(int unixPriority) {
+  if (unixPriority <= -20)
+    return REALTIME_PRIORITY_CLASS;
```

Comment:
I don't think this is desirable on Windows. I suggest you leave this out entirely.

Reply:
Done